### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: 'servers.default.yml'

### ISSUE THREAD

**Issue by solid-pixel** [Taxel/PlexTraktSync#1228]
### Confirmation

- [X] I have read the [README.md](https://github.com/Taxel/PlexTraktSync#readme) on the project homepage
- [X] I have checked if identical issue already exists

### The problem

Hello,

Every version after `0.23.14` (0.23.0, 0.23.1, 0.23.2) throws this error when I run any `plextraktsync` command.

There is indeed no `servers.default.yml` at the location mentioned in the error.

### Error trace / logs

```text
$ plextraktsync
[2022-11-27 00:35:32] INFO     PlexTraktSync [0.24.2]                                                                    
Traceback (most recent call last):
  File "/home/mrrobot/.local/bin/plextraktsync", line 8, in <module>
    sys.exit(cli())
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/click/core.py", line 1635, in invoke
    rv = super().invoke(ctx)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/plextraktsync/cli.py", line 53, in cli
    sync()
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/plextraktsync/cli.py", line 28, in wrap
    cmd(*args, **kwargs)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/plextraktsync/commands/sync.py", line 39, in sync
    w = factory.walker
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/plextraktsync/decorators/cached_property.py", line 23, in __get__
    value = obj.__dict__[self.func.__name__] = self.func(obj)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/plextraktsync/factory.py", line 123, in walker
    plex = self.plex_api
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/plextraktsync/decorators/cached_property.py", line 23, in __get__
    value = obj.__dict__[self.func.__name__] = self.func(obj)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/plextraktsync/factory.py", line 18, in plex_api
    server = self.plex_server
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/plextraktsync/decorators/cached_property.py", line 23, in __get__
    value = obj.__dict__[self.func.__name__] = self.func(obj)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/plextraktsync/factory.py", line 37, in plex_server
    server = self.server_config
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/plextraktsync/decorators/cached_property.py", line 23, in __get__
    value = obj.__dict__[self.func.__name__] = self.func(obj)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/plextraktsync/factory.py", line 55, in server_config
    server_config.load()
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/plextraktsync/config/ServerConfig.py", line 29, in load
    defaults = loader.load(default_servers_file)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/plextraktsync/config/ConfigLoader.py", line 5, in load
    return cls.load_yaml(path)
  File "/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/plextraktsync/config/ConfigLoader.py", line 45, in load_yaml
    with open(path, "r", encoding="utf-8") as fp:

FileNotFoundError: [Errno 2] No such file or directory: '/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/plextraktsync/servers.default.yml'
```

### Expected behavior

It should work by loading the server from the .env file. 
Unless something's changed with 0.24?

### Steps to reproduce the behavior

1. Update from 0.23.14 to 0.24.x
2. run `plextraktsync`

### Workarounds

Downgrade to 0.23.14

### Install method

pipx

### Config file contents

```yaml
# Config File: /home/mrrobot/.config/PlexTraktSync/config.yml
cache:
  path: /home/mrrobot/.cache/PlexTraktSync/trakt_cache
config:
  dotenv_override: true
excluded-libraries:
- X0R
logging:
  append: true
  console_time: true
  debug: false
  filename: plextraktsync.log
  filter: null
plex:
  timeout: 30
sync:
  plex_to_trakt:
    collection: true
    ratings: true
    watched_status: true
    watchlist: true
  trakt_to_plex:
    liked_lists: true
    ratings: true
    watched_status: true
    watchlist: true
    watchlist_as_playlist: false
watch:
  add_collection: false
  remove_collection: false
  scrobble_threshold: 80
  username_filter: true
xbmc-providers:
  movies: imdb
  shows: tvdb
```

### Version

0.24.2

### Python Version

3.7.3

### Operating System and Version

Linux polaris 4.19.0-18-amd64 #1 SMP Debian 4.19.208-1 (2021-09-29) x86_64 GNU/Linux

**Comment by solid-pixel** [comment]
Update:

I've downloaded https://github.com/Taxel/PlexTraktSync/blob/main/plextraktsync/servers.default.yml to the location mentioned in the error:

`/home1/mrrobot/.local/pipx/venvs/plextraktsync/lib/python3.7/site-packages/plextraktsync/servers.default.yml`

and now it runs successfully.

A file `servers.yml` has been created automatically in my config folder upon running `plextraktsync`, containing the server info probably grabbed from the .env file.

I guess the issue is that `servers.default.yml` is somehow missing from the python package.

**Comment by glensc** [comment]
Should be fixed in 0.24.3
